packages/lesswrong/components/ea-forum/users/DisplayNameWithMarkers.tsx

```diff
@@ -0,0 +1,55 @@
+import React from "react";
+import { registerComponent, Components } from "../../../lib/vulcan-lib";
+
+const styles = (_theme: ThemeType) => ({
+  tooltipPopper: {
+    maxWidth: 200,
+    textAlign: 'center'
+  },
+});
+
+/**
+ * Wrap certain special characters with a tooltip explaining them
+ */
+const DisplayNameWithMarkers = ({ name, classes }: { name: string; classes: ClassesType<typeof styles> }) => {
+  const { LWTooltip } = Components;
+
+  // Show a tooltip if they have the 🔸 or 🔹 at the end of their profile
```

Comment:
I feel like these emojis should be constants somewhere instead of littering the code with characters that can't easily be typed.